src/shared/services/filter.service.ts

```diff
@@ -5,20 +5,20 @@ import {
   ICalculatedProduct,
   IFilterOption,
   IFilterDefinition,
-} from '../models';
-import { ProductFilterOption } from '../enums';
-import { CategoryFacadeService, BrandFacadeService, FilterFacadeService } from '../services';
+  ProductFilterOption,
+  CategoryFacadeService,
+  BrandFacadeService,
+} from '@shared-module';
```

Comment:
Why did you change the imports here? It should remain relative and not use the alias.